src/ILAMB/Confrontation.py

```diff
@@ -144,13 +142,15 @@ def __init__(self,**keywords):
         self.plot_unit      = keywords.get("plot_unit",None)
         self.space_mean     = keywords.get("space_mean",True)
         self.relationships  = keywords.get("relationships",None)
+        self.sensitivities  = keywords.get("sensitivities",None) # YW
         self.keywords       = keywords
         self.extents        = np.asarray([[-90.,+90.],[-180.,+180.]])
         self.study_limits   = []
         
         # Make sure the source data exists
-
-        if not os.path.isfile(self.source):
+        try:
```

Comment:
Is there a reason that what I had wasn't working? I think the isfile is more portable across other systems?